include/ddsrouter/event/SignalHandler.hpp

```diff
@@ -120,6 +124,9 @@ class SignalHandler : public EventHandler<int>
     //! Unset for while process the signal handler routine.
```

Comment:
method `signal_handler_routine_` had an int as argument because it was the lambda for `signal`, and it was required.
Now it is not required to pass the int, so it could be obviated that it is the correct one.